.github/workflows/build-android.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       version: ${{ inputs.version }}
       version_file: ${{ inputs.version_file }}
       prefix: ${{ inputs.prefix }}
-    runs-on: macos-latest-xlarge
+    runs-on: ubuntu-latest
```

Comment:
We are changing this coz somehow CI is not working on Android correct?

Reply:
We are running out of disk space using a macOS runner

Reply:
Thanks for making this change.